sdk/test/trace/sampler_benchmark.cc

```diff
@@ -1,3 +1,4 @@
+#include "opentelemetry/exporters/memory/in_memory_span_exporter.h"
```

Comment:
@pyohannes This test is still unable to find my exporter. Could you take a look at my build and cmake file and see if there is something misconfigured? I'm using the existing exporters as reference, but you have more experience with this setup and might be able to see something I'm overlooking.

Reply:
This isn't very elegant at all, but let's try a relative include here:

```c
 #include "../../../exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter.h"
```

I think the problem you're seeing is related to a circular dependency (the sdk and its tests depend on the memory exporter, and the memory exporter in turn depends on the SDK). If the relative import doesn't work, we might have to change things in the global `CMakeLists.txt`.

Reply:
This workaround works for `./ci/run_docker.sh ./ci/do_ci.sh cmake.test`, but fails for bazel. When running one test vs another, it's requiring a separate set of imports. See below.

For tests:
```
// Import for CMake
// #include "../../../exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter.h"

// Import for Bazel
#include "opentelemetry/exporters/memory/in_memory_span_exporter.h"
```

And similarly for the exporter itself:
```
// Import for CMake
// #include "in_memory_span_data.h"

// Import for Bazel
#include "opentelemetry/exporters/memory/in_memory_span_data.h"
```

Reply:
Resolved